crates/bevy_dev_tools/src/ui_debug_overlay/mod.rs

```diff
@@ -219,7 +219,7 @@ fn outline_roots(
         // We skip ui in other windows that are not the primary one
         if let Some(camera_entity) = maybe_target_camera
             .map(|target| target.0)
-            .or(cam.default_ui_camera.get())
+            .or_else(|| cam.default_ui_camera.get())
```

Comment:
Not a fan of this change: which lint is this coming from?

Reply:
All of the closure-inducing changes are from [or_fun_call](https://rust-lang.github.io/rust-clippy/master/index.html#/or_fun_call)